### PR TITLE
Add tags to job list

### PR DESCRIPTION
**Pull request by eisuke**
retry of https://github.com/cookpad/kuroko2/pull/20 .
cc: @hogelog @cookpad/dev-infra

### CODE REVIEW

**Comment by adorechic** [comment]
seems ci has broken

File: app/views/kuroko2/job_definitions/_search_results.html.slim [hunk 1]
```diff
@@ -26,22 +26,25 @@
...
                 = link_to user.name, user_path(user)
                 br
-            td= first_line(definition.description)
             td
-              = link_to raw('<i class="fa fa-chevron-right"></i> View Details'), definition, class: 'btn btn-sm btn-default'
+              - definition.tags.each do |tag|
+                =link_to tag.name, job_definitions_path('tag': [tag.name]), class: 'label label-default spacer-right-3'
+              = first_line(definition.description)
+            td
+              = link_to raw('<i class="fa fa-chevron-right"></i> <span class="small">View Details</span>'), definition, class: 'btn btn-sm btn-default'
```
Comment (k0kubun) [inline app/views/kuroko2/job_definitions/_search_results.html.slim]: Can we use `= link_to definition, class: 'btn btn-sm btn-default' do ...`? Specifying `raw` by hand is dangerous.

Comment (eisuke) [inline app/views/kuroko2/job_definitions/_search_results.html.slim]: Fixed 37182b6.

But there are many `raw` functions in kuroko2 views 🤕

**Comment by k0kubun** [comment]
LGTM

**Pull request merged**